e2e/e2etests/test_crosschain_swap.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/zeta-chain/zetacore/x/crosschain/types"
 )
 
-func TestCrosschainSwap(r *runner.E2ERunner) {
+func TestCrosschainSwap(r *runner.E2ERunner, _ []string) {
```

Comment:
I see that there are values that can be passed as arguments for the swap. Having them hardcoded is fine but it is something to consider.